docs/facilitation/Remote.md

```diff
@@ -49,16 +49,54 @@ A good way to conclude the workshop is to bring everyone together again, and ask
 - Bring everyone together in the main session
 - Round rob through the mobs
 
-## General Facilitation tips
+## Tool
```

Comment:
Just a minor change here, I think it is better to put it in Plural 'Tools' instead of Tool